candidate_models/base_models/__init__.py

```diff
@@ -45,6 +45,7 @@ def init(identifier, preprocessing_type, image_size, net_name=None, labels_offse
         import tensorflow as tf
         from nets import nets_factory
 
+        tf.reset_default_graph()
```

Comment:
This should solve the issue with reloading tensorflow models: It resets the graph when another tensorflow model is called and this variable error doesn't happen any more

Reply:
ok got it. My worry is that if someone tries to retrieve two TF models at the same time, one of their graphs will be reset, leading to unexpected behavior (i.e. one model being unusable). But also not sure what we can do about that. I would at least warn the user if the graph is not empty, or at least let them know that we are resetting the graph and that this will kill any previous TF models in the same process.

Reply:
If somebody tries this, I would guess it fails because of tensorflow. I mean the error I'm solving comes from trying to load tensorflow models sequentially. Pretty sure parallel access is then even more complicated. After all we can't fix all problems. And since Python is not really parallel I think it's okay to require sequential model access.